js/avatarMovement.js

```diff
@@ -0,0 +1,148 @@
+
+//player A variables
+var playerAx = 150;
+var playerAy = 150;
+var playerAHeight = 50;
+var playerAWidth = 50;
+var playerAOrientation='E';
+
+//Player B variables
+var playerBx = 550;
+var playerBy = 550;
+var playerBHeight = 50;
+var playerBWidth = 50;
+var playerBOrientation='W';
+
+//create players
+var playerA = drawPlayer(playerAx, playerAy, playerAHeight, playerAWidth, 'pink', 'playerA', playerAOrientation);
+var playerB = drawPlayer(playerBx, playerBy, playerBHeight, playerBWidth, 'blue', 'playerB', playerBOrientation);
+
+//gameplay variables
+var distance = 1000;
+var time = 3000;
+
+
+function drawPlayer (x, y, h, w, fill, id, orientation) {
+    var rect = document.createElementNS("http://www.w3.org/2000/svg", "rect");
+    rect.setAttributeNS(null, 'x', x);
+    rect.setAttributeNS(null, 'y', y);
+    rect.setAttributeNS(null, 'height', h);
+    rect.setAttributeNS(null, 'width', w);
+    rect.setAttributeNS(null, 'fill', fill);
+    rect.setAttributeNS(null, 'id', id);
+    rect.setAttributeNS(null, 'orientation',orientation)
+    document.getElementById('svgOne').appendChild(rect);
+    return document.getElementById(id);
+}
+
+
+//  <-  37 = left arrow,  39 = right arrow , 38 = up arrow, 40 = down arrow
+//  <-  65 = A (left) ,  68 = D (right)  , 87 = W (up) , 83 = S (down)
+function keyDownHandler(e) {
+    $playerA=$(playerA);
+    $playerB=$(playerB);
+    switch(e.keyCode) {
+        case 37:
```

Comment:
does this need playerAOrientation = 'W'?

Reply:
done

Reply:
updated